CONTRIBUTING.md

```diff
@@ -122,6 +122,48 @@ branch. Steps:
    git push -u origin my-feature
    ```
 
+## Local Developer Setup
+1. Make sure you have a compatible version of Python 3 installed
+2. From the project root, create a virtual environment and install all development dependencies
+
+   **Bash**
+    ```bash
+   # Create virtual environment
+   python3 -m venv venv
+   
+   # Activate the virtual environment
+   source venv/bin/activate
+   
+   # Install development dependencies
+   pip install -U pip setuptools
+   pip install -r requirements.txt
+   ```
+   **Windows (CMD)**
+    ```cmd
+   rem Create virtual environment
+   python3 -m venv venv
+   
+   rem Activate the virtual environment
+   .\venv\Scripts\activate.bat
+   
+   rem Install development dependencies
+   pip install -U pip setuptools
+   pip install -r requirements.txt
+   ```
+   **Windows (PowerShell)**
```

Comment:
I feel like a jerk even bringing it up, but what about powershell on linux?  (you can tell me to stfu ;D)

Reply:
Updated to remove OS restriction on PowerShell script.